package.json

```diff
@@ -90,5 +90,8 @@
     "q": "1.4.1",
     "u2f-api-polyfill": "0.4.1",
     "underscore": "1.8.3"
+  },
+  "optionalDependencies": {
+    "fsevents": "*"
```

Comment:
Apparently since the fsevents was part of the npm-shrinkwrap, it started failing hard on npm install unless I added this in. I guess the other option would be to remove it from the npm-shrinkwrap? I wasn't sure which was the better way to go.